docs-en/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -390,13 +390,13 @@ Building a linked list involves two steps: initializing each node object and the
     n3.next = &n4;
     ```
 
-An array is a single variable, such as the array `nums` containing elements `nums[0]`, `nums[1]`, etc., while a linked list is composed of multiple independent node objects. **We usually refer to the linked list by its head node**, as in the linked list `n0` in the above code.
+An array is one cohesive variable, for instance, the array nums includes elements like nums[0], nums[1], and so on, whereas a linked list is made up of several distinct node objects.  **We typically identify a linked list by its head node**, for example, the linked list in the previous code snippet is referred to as `n0`.
```

Comment:
please delete the extra space leading "**We".
"identify a linked list" -> "refer to a linked list".